Packages/Status/Sources/Status/Row/Subviews/StatusRowCardView.swift

```diff
@@ -114,4 +82,89 @@ public struct StatusRowCardView: View {
     }
     .buttonStyle(.plain)
   }
+
+  @MainActor
```

Comment:
FYI you don't need `MainActor` but you should use `ViewBuilder` so you can remove the `Group`

Reply:
I do need to learn how to use that…  :)